### ISSUE TITLE
[High contrast] AutoSuggestBox is not styled properly

### ISSUE THREAD

**Issue by jayrenn** [winjs/winjs#1347]
* There should be an outline around the search results.
* The border and text of the search box are the wrong color.

What I see:
 ![image](https://cloud.githubusercontent.com/assets/7266075/9261976/8018d30e-41c6-11e5-8372-310057b9caf3.png)

What it should be:
![image](https://cloud.githubusercontent.com/assets/7266075/9261980/86a36de2-41c6-11e5-8b36-cfda349470af.png)
